### PR TITLE
LL-719 Add Apps page

### PR DESCRIPTION
**Pull request by h-kanazawa**
#### Which issue does this close?

Closes https://github.com/LearningLocker/enterprise/issues/884
Closes https://ht2labs.myjetbrains.com/youtrack/issue/LL-719

#### Describe your changes providing screenshots of UI changes if necessary.

- Add "App Store" to side nav
- Add 3 apps to app page.

### CODE REVIEW

File: ui/src/containers/SideNav/index.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ class SideNav extends Component {
           {canViewOrganisations && this.renderSettingsLink('suborgs', 'Organisations')}
           {canViewClients && this.renderSettingsLink('clients', 'Clients')}
           {canViewRoles && this.renderSettingsLink('roles', 'Roles')}
+          {this.renderSettingsLink('apps', 'App Store')}
```
Comment (ryansmith94) [inline ui/src/containers/SideNav/index.js]:
```suggestion
          {this.renderSettingsLink('apps', 'Apps')}
```

**Comment by Ian247** [comment]
Text for cards

Curatr: xAPI enabled, cutting edge Learning experience platform that is designed to work hand-in-hand with Learning Locker.

Moodle: Widely supported open source learning platform supported by a custom xAPI plugin.

**Comment by Ian247** [comment]
Update to fit size:
Cutting-edge Learning experience platform that is designed to work hand-in-hand with Learning Locker.

File: ui/src/containers/Apps/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  {
+    link: 'https://withsparks.com/',
+    title: 'Sparks',
+    description: 'Use statements to trigger follow on actions, deliver personalized feedback and automate workflow.',
+    icon: icons.sparks,
+    icon2x: icons.sparks2x,
+  },
+  {
+    link: 'https://ht2ltd.zendesk.com/hc/en-us/articles/208166165-xAPI-in-Curatr',
+    title: 'Curatr',
```
Comment (ryansmith94) [inline ui/src/containers/Apps/index.js]:
```suggestion
    title: 'Curatr LXP',
```

**Comment by asahd** [comment]
**NOTE: DON'T MERGE THIS YET:**
https://github.com/LearningLocker/enterprise/issues/884#issuecomment-473355335

**Comment by h-kanazawa** [comment]
@ryansmith94 @asahd I removed card outline in "Apps"

**Pull request merged**